src/effects/effectmanifest.h

```diff
@@ -82,6 +83,14 @@ class EffectManifest final {
         m_isMixingEQ = value;
     }
 
+    virtual const bool& isWaveformChangingEQ() const {
+        return m_isWaveformChangingEQ;
```

Comment:
> If we have a EQ, that for example had a button that effects the high frequencies, it can have a property on this parameter, that it changes the high gain.

But we do not have such an EQ effect that does that without effecting the mids and lows. If we add one in the future it would not be too difficult to add that capability, but there is no need for that here.